apps/src/code-studio/progressRedux.ts

```diff
@@ -83,6 +83,7 @@ export interface ProgressState {
   courseVersionId: number | undefined;
   unitDescription: string | undefined;
   unitStudentDescription: string | undefined;
+  unitHasUnnumberedLessons: boolean | undefined;
```

Comment:
Couldn't we just default to `false` and make the type simpler here?

```suggestion
  unitHasUnnumberedLessons: boolean;
```